### ISSUE TITLE
Map Zoom via scroll does not work

### ISSUE THREAD

**Issue by pseabury** [arnaudleclerc/AzureMapsControl.Components#27]
1.) Create a default Server-side Blazor App (.NET 5)
2.) Add a Map to the page (along with style for map size)
3.) Run app and observe that UserInteraction is enabled, and all functions work except for mouse wheel zoom.

```
#map {
    height: 400px;
}
```

```
<h1> Map </h1>
<AzureMap Id="map"
          CameraOptions="new CameraOptions { Center= new AzureMapsControl.Components.Atlas.Position(-83.176687, 42.205024), Zoom= 9 }"
          StyleOptions="MapStyleOptions"
          EventActivationFlags="MapEventActivationFlags
                                .None()
                                .Enable(MapEventType.Ready)"
          OnReady="OnMapReady" />
```

```
    public StyleOptions MapStyleOptions = new StyleOptions
    {
        Style = "grayscale_dark",
        ShowLogo = false,
        ShowFeedbackLink = false
    };
    public async Task OnMapReady(MapEventArgs events)
    {
    }
```

There are no errors in Browser console or exceptions thrown in server-side code.

![image](https://user-images.githubusercontent.com/1918526/105608229-62c52000-5d68-11eb-8f03-22ea16d8092a.png)

**Comment by pseabury** [comment]
Had to manually set the UserInteractionOptions.WheelZoomRate to 1.  The defaults for userInteraction are all set the true, but the WheelZoomRate is set to 0 by default.

**Comment by arnaudleclerc** [comment]
You are totally right, it should use the default values of the azure-maps library. I will publish a fix ASAP

**Comment by arnaudleclerc** [comment]
It should be fixed on the upcoming v0.9.0. The user interactions will be activated by default (as of now) and their default values will be the ones defined by the azure-maps-control library. You should check if the default value of the `WheelZoomRate` suits you.
